src/components/Article/index.tsx

```diff
@@ -55,6 +57,7 @@ export default class Article extends React.Component<ArticleProps> {
     return (
       <article
         className={css`
+          ${style};
```

Comment:
Not really a fan of this style – can we use `this.props.style` directly?
```suggestion
          ${this.props.style};
```